db/models/order.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const {STRING, INTEGER, DOUBLE} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  orderNumber: STRING,
+  status: STRING,
+  qty: INTEGER,
+  price: DOUBLE
+}, {
+  getterMethods: {
+    getOrdNum: () =>
+      `SS-${new Date().toISOString().slice(0, 10)}-${Math.floor(Math.random()*100)+400}`
```

Comment:
The getterMethod is going to return a different order number every time, because you are using Math.random.